tests/e2e/specs/in-game/reconnecting.spec.js

```diff
@@ -699,3 +699,219 @@ describe('Reauthenticating in game', () => {
     cy.get('#deck').should('be.visible');
   });
 });
+
+describe('Reconnecting after game is over', () => {
+  beforeEach(() => {
+    cy.setupGameAsP0();
+  });
+  
+  it('Dialogs persist after refreshing when game is over by conceded and opponent request rematch', () => {
+    cy.concedeOpponent();
+    cy.get('[data-cy=game-over-dialog]').should('be.visible');
+    cy.url().then((url) => {
+      const oldGameId = Number(url.split('/').pop());
+      cy.rematchOpponent({ gameId: oldGameId, rematch: true });
+    });
+    cy.get('[data-cy=lobby-ready-card]').should('be.visible');
+    cy.reload();
+    cy.get('[data-cy=game-over-dialog]').should('be.visible');
+    cy.get('[data-cy=match-score-counter-wins]').should('contain', 'W: 1');
+    cy.get('[data-cy=lobby-ready-card]').should('exist');
```

Comment:
We can be more specific for these assertions to check that the card shows the correct player is ready. See `tests/e2e/specs/in-game/game-over/rematch.spec.js` for examples:

```
  cy.get('[data-cy=opponent-rematch-indicator]')
    .find('[data-cy="lobby-card-container"]')
    .should('have.class', 'ready');
```

Reply:
Then while we're at it we can assert the 'my' card appears and is not ready